src/logger/src/metrics.rs

```diff
@@ -185,7 +190,7 @@ impl fmt::Display for MetricsError {
 /// or an arbitrary amount of units.
 // This trait helps with writing less code. It has to be in scope (via an use directive) in order
```

Comment:
I think we should also update this comment. Currently, it states the general purpose of traits, which is redundant. It should be similar to the comment associated to pub trait `StoreMetric`, stating the use case